src/Sentry/Laravel/Features/QueueIntegration.php

```diff
@@ -80,7 +80,6 @@ public function onBoot(Dispatcher $events): void
 
                 if ($payload !== null) {
                     $payload[self::QUEUE_PAYLOAD_BAGGAGE_DATA] = getBaggage();
-                    $payload[self::QUEUE_PAYLOAD_TRACE_PARENT_DATA] = getTraceparent();
```

Comment:
Was it intentional to remove this line? Seems unrelated to w3ctraceparent..

Reply:
Thanks, https://github.com/getsentry/sentry-laravel/pull/995